tests/ixia/test_ixia_traffic_restpy.py

```diff
@@ -0,0 +1,183 @@
+###############################################################################
+# This test cases demonstrates:
+#   * All the fixtures required for running ixia script (please see the
+#     arguments of the test function)
+#   * How Ixia chassis card/ports are addressed
+#   * How you can configure/control ixia devices, start traffic and collect
+#     statistics using REST API
+#   * This simple sanity test cases can be used to check if testbed setup
+#     is correct or not - since it prints a lot of testbed data
+###############################################################################
+
+import logging
+import time
+import pytest
+import pprint
+from common.utilities import wait_until
+from common.fixtures.conn_graph_facts import conn_graph_facts
+from common.platform.interface_utils import check_interface_information
+from common.platform.daemon_utils import check_pmon_daemon_status
+from common.reboot import *
+from common.platform.device_utils import fanout_switch_port_lookup
+from common.helpers import assertions
+from ixnetwork_restpy import SessionAssistant, Files
+
+from common.ixia_fixtures import ixia_api_serv_ip, ixia_api_serv_user,\
+     ixia_api_serv_passwd, ixia_dev, ixia_api_serv_port, ixia_api_serv_session_id
+
+from common.ixia_helpers import get_neigh_ixia_mgmt_ip, get_neigh_ixia_card,\
+     get_neigh_ixia_port
+
+import time
+
+#pytestmark = [pytest.mark.disable_loganalyzer]
```

Comment:
cleanup. remove

Reply:
I have addressed this - the commented line is removed